src/components/home/elements/SendToken.vue

```diff
@@ -540,33 +558,21 @@ export default {
                 timestamp: Date.now(),
                 attachment: transaction.prepareSendAttachment(this.coldAttachment),
                 contractId: this.contractId,
-                functionId: this.functionIndex,
+                functionId: this.isSplit ? SEND_FUNCIDX_SPLIT : SEND_FUNCIDX,
                 function: transaction.prepareSend(this.coldRecipient, BigNumber(this.coldAmount).multipliedBy(this.tokenUnity)),
                 functionExplain: 'send ' + this.coldAmount + ' token to ' + this.coldRecipient
             }
-        },
-        isValidAttachment() {
-            if (!this.attachment) {
-                return void 0
-            }
-            return common.getLength(this.attachment) <= TRANSFER_ATTACHMENT_BYTE_LIMIT
-        },
-        isValidColdAttachment() {
-            if (!this.coldAttachment) {
-                return void 0
-            }
-            return common.getLength(this.coldAttachment) <= TRANSFER_ATTACHMENT_BYTE_LIMIT
         }
     },
     methods: {
+        isValidAttachment(attachment) {
```

Comment:
We'd better keep "if"
```
if (!this.attachment) {		
     return void 0		
}
```
Or it will show "green tick" if empty attachment.

Expected UX principle:
Valid attachment --> Tick
Invalid attachment --> Cross
No attachment --> No symbol